agent/engine/execcmd/manager_unsupported.go

```diff
@@ -1,4 +1,4 @@
-// +build !linux,!windows
+// +build !linux,!unit
```

Comment:
`!windows`?

Reply:
If it's windows I do want this code so that all exec cmd stuff is a no-op

Reply:
The reason I removed that was that I had a separate `manager_windows.go` file, but I realized that was not needed and I can just have this one